src/AutoBuild.jl

```diff
@@ -1425,6 +1425,29 @@ const uuid_package = UUID("cfb74b52-ec16-5bb7-a574-95d9e393895e")
 # For even more interesting historical reasons, we append an extra
 # "_jll" to the name of the new package before computing its UUID.
 jll_uuid(name) = bb_specific_uuid5(uuid_package, "$(name)_jll")
+
+function find_uuid(ctx, pkg)
+    if Pkg.Types.has_uuid(pkg)
+        return pkg.uuid
+    end
+    depname = getname(pkg)
+    @static if VERSION >= v"1.7"
+        uuids = Pkg.Types.registered_uuids(ctx.registries, depname)
+    else
+        uuids = Pkg.Types.registered_uuids(ctx, depname)
+    end
+    if isempty(uuids)
+        return nothing
```

Comment:
I was thinking of using `jll_uuid` here, but maybe it's better to have this function returning the real UUID, if any, only.

Reply:
Yeah the first version did that, but then I was like: Maybe I want to use that function elsewhere xD